cgmanifest.json

```diff
@@ -50,16 +52,17 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "2021-06-01",
+          "commitHash": "4244cd1cb492fa1d10986ec67f862964c073f844",
           "repositoryUrl": "https://github.com/google/re2.git"
-        }
+        },
+        "comments": "2021-06-01"
       }
     },
     {
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "release-1.11.0",
+          "commitHash": "9406a60c7839052e4944ea4dbc8344762a89f9bd",
```

Comment:
Not sure what tag this commitHash matches as there's no comment in the cmake file. 

https://github.com/microsoft/onnxruntime-extensions/blob/a1e342acb94c4b28c74c38e5b3a235824edb98fc/cmake/externals/googletest.cmake#L1-L5

The commit id doesn't match release-1.11.0. The readme.md in the googletest source says it's release-1.12.0, but the commit id for that is different.

https://github.com/google/googletest/releases/tag/release-1.12.0

Reply:
Yes, it was upgraded due to conflict with AzureOP build in CI pipeline.